### DISCUSSION TITLE
Example of `yarn nx run my-app-e2e:e2e --parallel` with GitHub Actions

### DISCUSSION THREAD

**Discussion by ghost** [nrwl/nx#7067] [General]
Hi everyone!

I was wondering if anyone has an example of running `yarn nx run lender-portal-e2e:e2e --parallel --key=XXX --record --ci --headless --exit` with cypress in parallel on github actions.

I'm having a hard time getting the tests to run in parallel with the Cypress dashboard. 

Im running a command in the matrix, but spawns the whole test suite in each of those containers instead of splitting the tests up

Thank you in advace

**Comment by smasala** [comment]
I'd also be interested in finding out how you can run the affected e2e tests in parallel when passing the `-- -parallel` down to cypress https://github.com/nrwl/nx/issues/1604#issuecomment-652618860
